### ISSUE TITLE
FORCE_NG_UPDATE=true ng update does not work with workspace version 2

### ISSUE THREAD

**Issue by minijus** [nrwl/nx#8130]
## Current Behavior
`FORCE_NG_UPDATE=true ng update` is broken after migration to project based configurations.
It breaks because version in `angular.json` is bumped to 2 and Angular CLI does version validation:
https://github.com/angular/angular-cli/blob/master/packages/angular_devkit/core/src/workspace/json/reader.ts#L53-L55

## Expected Behavior
It should still be possible to invoke Angular CLI (with FORCE env option) when using nx workspace version 2 (with project based configurations).

## Steps to Reproduce

- Change `version` inside `angular.json` from 1 to 2 (no need to run full generator to migrate to project based configs).
- Run `FORCE_NG_UPDATE=true ng update @angular/core`

![image](https://user-images.githubusercontent.com/3633549/145726812-68aa4aea-4ef2-4b2b-b9df-21427853a65a.png)

`FORCE_NG_UPDATE` would not be needed if `packageGroup` metadata would be respected in the same way as it is done inside `ng update` (#4575).

### Environment
```
  NX  Report complete - copy this into the issue template

  Node : 16.13.1
  OS   : darwin x64
  npm  : 8.3.0
  
  nx : 13.3.1
  @nrwl/angular : 13.3.1
  @nrwl/cli : 13.3.1
  @nrwl/cypress : 13.3.1
  @nrwl/devkit : 13.3.1
  @nrwl/eslint-plugin-nx : 13.3.1
  @nrwl/express : undefined
  @nrwl/jest : 13.3.1
  @nrwl/linter : 13.3.1
  @nrwl/nest : undefined
  @nrwl/next : undefined
  @nrwl/node : undefined
  @nrwl/nx-cloud : undefined
  @nrwl/react : undefined
  @nrwl/react-native : undefined
  @nrwl/schematics : undefined
  @nrwl/tao : 13.3.1
  @nrwl/web : undefined
  @nrwl/workspace : 13.3.1
  @nrwl/storybook : 13.3.1
  @nrwl/gatsby : undefined
  typescript : 4.4.4
  rxjs : 7.4.0
  ---------------------------------------
  Community plugins:
         @angular/animations: 13.1.0
         @angular/common: 13.1.0
         @angular/compiler: 13.1.0
         @angular/core: 13.1.0
         @angular/forms: 13.1.0
         @angular/platform-browser: 13.1.0
         @angular/platform-browser-dynamic: 13.1.0
         @angular/router: 13.1.0
         @angular-devkit/build-angular: 13.0.4
         @angular/cli: 13.0.4
         @angular/compiler-cli: 13.1.0
         @angular/language-service: 13.1.0
```

**Comment by Coly010** [comment]
@minijus I'm curious what the use case here is? 

If you run `ng migrate` instead of `ng update` do you not get the same expected results? 
We still run any migration that Angular has created for `ng update` when `ng migrate` is run. 

I'm curious as to what the unexpected behaviour is?

And to further that, Angular has set the constraint of v1 workspaces. It's their code that ends up failing, so there's not a lot that we can do in this regard. This is partially why we recommend `ng migrate` because we make sure to still run any migrations provided from `angular`.

**Comment by minijus** [comment]
@Coly010 The issue is that `nx migrate` does not respect `packageGroup` metadata (see: #4575). As original issue description says, this fallback to `ng update` would not be needed if `nx migrate` would take `packageGroup` into considerations.

I see your point about Angular failing, but I do not fully agree with it. It is `nx` that bumps the version breaking the fallback that worked before.

**Comment by leosvelperez** [comment]
This issue was fixed on https://github.com/nrwl/nx/pull/8993 and released in [v13.8.2](https://github.com/nrwl/nx/releases/tag/13.8.2).
